BSP/Simulator/DataGeneration/simulate.py

```diff
@@ -23,6 +24,7 @@
 CANbox = None   #box that displays the CAN messages
 lights_names = ['EXTRA', 'CAN', 'WDOG', 'UVOLT', 'OVOLT', 'OTEMP', 'OCURR', 'RUN', 'FAULT']
 frequency = None
+maxEEPROMAddress = 0x1FFFF 
```

Comment:
It has just occurred to me that I converted from decimal to hex incorrectly, and 16KB would give a maximum address of 0x3FFF